docs/api/session.md

```diff
@@ -690,6 +690,12 @@ The `proxyBypassRules` is a comma separated list of rules described below:
    Match local addresses. The meaning of `<local>` is whether the
    host matches one of: "127.0.0.1", "::1", "localhost".
 
+#### `ses.resolveHost(host)`
+
+* `host` string - Hostname to resolve.
+
+Returns `Promise<string[]>` - Resolves with the resolved IP addresses for the `host`.
```

Comment:
Is `string[]` the right return type for the endpoints here? I feel like there might be a possibility that we want to extend this API to return more data in future, e.g. i notice this discards all the data from [ConnectionEndpointMetadata](https://source.chromium.org/chromium/chromium/src/+/main:net/dns/public/host_resolver_results.h;l=20;drc=8101d6d81854e4f437e0f03f77694a01aaa8df7a). I'm not sure we care much about exposing that now, but I can imagine in future we might want to return more data here, and a `string` won't let us do that.

Perhaps either `Promise<{address: string}[]>` or `Promise<{addresses: string[]}>`?

Reply:
Agree. I'll do it in a bit.

Reply:
Done.